static/js/file_select.js

```diff
@@ -210,15 +185,61 @@ function populateTable(data) {
     }
 }
 
+
+function updateData(coords, change){
+    var infos = coords.split(',');
+    var field = tableLabels[infos[1]];
+    var entry = tableData[infos[0]];
+
+    if(field.includes("mineral")){
+        field = entry.hasOwnProperty('mineral') ? 'mineral' : 'minerals';
+    }
+    console.log("field, ", field);
+    if(field==="location_coords"){
+        var changes = change.split(',');
+        entry['latitude'] = parseFloat(changes[0]);
+        entry['longitude'] = parseFloat(changes[1]);
+    }else if(field==="amount"){
+        field = tableLabels[parseInt(infos[1])-1];
+        entry[field]["amount"] = change;
+    }else if(Array.isArray(entry[field])){
+        updateArray(entry,field,change);
+    }else {
+        entry[field] = change;
+    }
+}
+
+function updateArray(entry, field, change) {
+    var arr = change.split(',');
+    for(var i =0; i < arr.length;i++){
+            arr[i] = {"amount": 0, "name": arr[i]};
+    }
+}
+
+function createBanner(statusCode){
+    if (statusCode >=200 && statusCode < 300 ) {
+        var banner = document.getElementById("msgbanner");
+        banner.style.color = 'green';
+        banner.innerHTML = "No errors found!  Rows displayed below have been inserted into the database."
+        return false;
+    }
+    var banner = document.getElementById("msgbanner");
+    banner.style.color = 'red';
+    banner.innerHTML = "Please fix the highlighted area and resubmit."
+    return true;
+}
```

Comment:
Awesome work breaking out functions to make code clearer--something I should have done in the first place